src/core/pyproject.toml

```diff
@@ -0,0 +1,61 @@
+[build-system]
+build-backend = "poetry.core.masonry.api"
+requires = [
+  "poetry-core",
+]
+
+[tool.poetry]
+name = "trulens-core"
+version = "1.0.1a0"
+description = "Library to systematically track and evaluate LLM based applications."
+authors = [
+  "Snowflake Inc. <ml-observability-wg-dl@snowflake.com>",
+]
+license = "MIT"
+readme = "README.md"
+packages = [
+  { include = "trulens" },
+]
+homepage = "https://trulens.org/"
+documentation = "https://trulens.org/trulens/getting_started/"
+repository = "https://github.com/truera/trulens"
+classifiers = [
+  "Programming Language :: Python :: 3",
+  "Operating System :: OS Independent",
+  "Development Status :: 3 - Alpha",
+  "License :: OSI Approved :: MIT License",
+]
+
+[tool.poetry.dependencies]
+python = "^3.9"
+numpy = "^1.23"
+munch = "^3.0"
+dill = "^0.3"
+tqdm = "^4.66"
+typing_extensions = "^4.9"
+pydantic = "^2"
+merkle-json = "^1.0"
+sqlalchemy = "^2.0"
+alembic = "^1.11"
+nest-asyncio = "^1.5"
+python-dotenv = "^1.0"
+
+snowflake-core = { version = "^0.9.0", python = "<3.12", optional = true }
+snowflake-connector-python = { version = "^3.11.0", python = "<3.12", optional = true }
+snowflake-snowpark-python = { version = "^1.20.0", python = "<3.12", optional = true }
+snowflake-sqlalchemy = { version = "^1.6", python = "<3.12", optional = true }
```

Comment:
We should repackage them as part of `trulens-snowflake` and not part of `trulens-core`?

Reply:
Seems like you put this as an extra in poetry. I guess that is alright.

Reply:
How are we thinking of packaging snowflake integrations? We support snowflakedb connection details in `Tru()` but not all users need to integrate with Snowflake. Perhaps `trulens-snowflake` with a `SnowflakeTru()` may be more appropriate

Reply:
Maybe something like `trulens-logging-snowflake` to disambiguate from `trulens-providers-cortex`